packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -2677,6 +3087,54 @@ mod tests {
         asset_router
     }
 
+    #[fixture]
+    fn long_asset_router() -> AssetRouter<'static> {
+        let mut asset_router = AssetRouter::default();
+
+        let assets = vec![
+            Asset::new("long_asset_one_chunk", long_asset_body(ASSET_CHUNK_SIZE)),
+            Asset::new(
+                "long_asset_two_chunks",
+                long_asset_body(TWO_CHUNKS_ASSET_LEN),
```

Comment:
What about passing the path here instead of the length? So the decision on the asset path and config path is in the same place.

Reply:
Avoiding some repetition/dependencies in the setup is a good idea, so yeah I've refactored the code slightly.  I also randomized the asset body, to have a more robust test. PTAL.